Frontend/app/user/registration/registration.component.ts

```diff
@@ -14,6 +14,7 @@ export class RegistrationComponent {
   mentiiConfig = new MentiiConfig();
   submitInProgress = false;
   regSuccess = false;
+  isLoading = false;
```

Comment:
Is there a difference between 'isLoading' and 'submitInProgress'? They have awfully similar names